src/Apps/Sell/Components/UploadPhotosForm.tsx

```diff
@@ -131,6 +131,7 @@ export const UploadPhotosForm: React.FC = () => {
       border="1px dashed"
       borderColor="black30"
       padding={4}
+      hideBoxOnMobile
```

Comment:
To keep everything as simple as possible.

Reply:
Yeah, we use this component on other flows :/  
Old condign flow, but most importantly -- my collection add/edit artwork form

Reply:
Ok, I suppose we could introduce the new design in both places as well. This would reduce the complexity in the code (no more prop that does something only on small screen sizes = difficult to test) and we would achieve more consistency among all the artwork forms.

hideBoxOnMobile

Reply:
We would just need some extra padding above the button probably in the My Collection artwork form.

Reply:
Sounds good 👍

Reply:
And also I should remove Box component for xs Media, <input /> is enough